_docs/welcome/notes/0.4.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Istio 0.4
+
+order: 97
+
+layout: docs
+type: markdown
+---
+
+## General
+
+This release has only got a few weeks worth of changes, as we stabilize our monthly release process. 
+In addition to the usual pile of bug fixes and perf improvements, this release includes:
+
+- **Cloud Foundry**. Added support for the [Cloud Foundry](https://www.cloudfoundry.org) platform to Pilot, making it
+possible for Pilot to discover CF services and service instances. This represents the strict minimum needed to
+get things running, additional work remains in this area to flesh things out.
+
+- **Circonus**. Mixer now includes an adapter for the [Circonus](https://www.circonus.com) analytics and monitoring platform.
+
+- **Pilot Metrics**. Pilot now collects metrics for diagnostics.
+
+- **Helm Charts**. We now provide helm charts to install Istio. [DOC LINK?]
+
+- **Enhanced Attribute Expressions**. Mixer's expression language gained a few new functions. [DOC LINK?]
```

Comment:
This is very terse. Can we highlight some of the specifics?

Reply:
OK.